### ISSUE TITLE
How do people find Concepts; how useful are Concepts

### ISSUE THREAD

**Issue by taceybadgerbrook** [wellcomecollection/wellcomecollection.org#10196]
**What is the problem to solve?**
Less than 10% of people who search view Concepts pages. Why?

**What are the objectives?**
Look at how to people get to concepts from site pages using source field which will point to the page before. 

**What research technique is required?**
Python

**Comment by taceybadgerbrook** [comment]
## Types of concept pages and what this looks like in practice

### Organisation 
example: https://wellcomecollection.org/concepts/ra8scmeh
maps to subjects even though Organisation displayed at top

Concepts page looks like:
<img width="200" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/a13a52b7-ec76-4da6-810d-33d633b368a3">

Link on works page looks like:
<img width="600" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/a004d9a4-fa0f-4c06-8102-f8314af92fa2">

**Comment by taceybadgerbrook** [comment]
## Place

example: https://wellcomecollection.org/concepts/x7y8pju7
maps to subjects even though Place displayed at top

Concepts page looks like:
<img width="200" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/c4d59324-9c1a-4e98-a80f-da0d1b119fc3">

Link on works page looks like:
<img width="231" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/3ced4651-78c6-4aca-af48-3fb6a79ecbf2">

## Concept

example: https://wellcomecollection.org/concepts/j9hu3f9u
maps to subjects contributors genres even though Organisation displayed at top

Concepts page looks like:
<img width="200" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/6ac964b3-9989-4f28-86ae-c7d6ea3de424">

Link on works page looks like:
<img width="200" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/cefa8474-dcd0-491b-8ad3-4407c1ed6296">

**Comment by taceybadgerbrook** [comment]
## Person

example: https://wellcomecollection.org/concepts/fppbxh8b

Concepts page looks like:
<img width="200" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/cb19dcf7-d75d-461e-ac78-aa1a882c2f07">

Link on works page looks like:
<img width="732" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/2a570c8b-5700-4708-aa31-518764565d4c">



## Type/technique

example: https://wellcomecollection.org/concepts/tbeys2vz

Concepts page looks like:
<img width="200" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/8c89f33d-d85c-475d-9e09-2d2fadc2213f">

Link on works page looks like:
<img width="196" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/d554fff4-b58e-4749-b78a-1d100cf2bf21">

**Comment by taceybadgerbrook** [comment]
## Place - the difference between what the link on the works page promises and what is displayed once clicked could confuse users.

example: https://wellcomecollection.org/concepts/x7y8pju7
maps to subjects even though Place displayed at top

Concepts page looks like:
<img width="200" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/c4d59324-9c1a-4e98-a80f-da0d1b119fc3">

Link on works page looks like:
<img width="231" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/3ced4651-78c6-4aca-af48-3fb6a79ecbf2">

**Comment by taceybadgerbrook** [comment]
@LFBaily I analysed 100,000 Elasticsearch records 30-31/1/24 identifying 2,771 users who had viewed concepts pages. Counting actions as search, works page view, item view (including pagination), modal image view, image item view, exhibition view and story view, I filtered this to come up with a dataset comprising the concepts page view and each subsequent action. 

2,320 users (84% of all user who viewed a concepts page) ended their session after viewing a concepts page, ie they did not click on any of the links provided. 216 users (8% of all users who viewed a concepts page) engaged in one more action and then concluded their session. 

Here's a summary table for numbers of actions 1-10:
<img width="172" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/e95e3181-b38f-4c2e-bc2b-b07437255d15">

**Comment by taceybadgerbrook** [comment]
Here's a graph of counts of actions per user:

<img width="566" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/5550f6a4-a3e6-4560-b73d-8bb12968d5e2">

**Comment by taceybadgerbrook** [comment]
I think the next piece of analysis will be digging into which kinds of Concepts pages users left directly after viewing.

**Comment by taceybadgerbrook** [comment]
I had expected that concepts appearing as "Subjects" on works pages but displaying as "Organisation" or "Place" on concepts pages would be the kinds of pages that confused users leading them to leave the site. However, further analysis into users ending their sessions with a concept page showed no such pattern. 

The concept that was viewed most frequently as the last in a session was https://wellcomecollection.org/concepts/t9kz4e5p (10 times on 31/1/2024-1/2/2024.) This concept is show as a "Contributor" and displayed as "Person" on the concepts page. 

GA4 shows that for the month of January 2024, the concept page https://wellcomecollection.org/concepts/t9kz4e5p was viewed 340 times and had 340 exits. Looking more closely at this page, all of the links are to articles in Spanish but I wonder if this is actually part of end to end testing?

**Comment by taceybadgerbrook** [comment]
GA4 shows that for the month of January 2024, the concept page https://wellcomecollection.org/concepts/t9kz4e5p was viewed 340 times and had 340 exits. Looking more closely at this page, all of the links are to articles in Spanish but I wonder if this is actually part of end to end testing?

**Comment by rcantin-w** [comment]
It isn't used in e2es; you can see them here if of interest https://github.com/wellcomecollection/wellcomecollection.org/blob/main/playwright/test/concept.test.ts
(line 13, 20, 27 and 36)

**Comment by taceybadgerbrook** [comment]
Visualising user journeys from concepts page view and onwards:
- anonymousId (to sort by)
- @timestamp
- page.path

**Comment by taceybadgerbrook** [comment]
Of 200,000 Elasticsearch records analysed during the period 3/3/24 - 6/3/24, 7,523 users viewed concepts pages out of a total of 49,617 (that's 15% of total users.)

**Comment by taceybadgerbrook** [comment]
Following recoding of the source field, I can now link clicks on concepts pages links to works pages and item views as well as clicks on concepts pages links to modal views and image item views. Analysis of 40,000 records during 19/3/24 contained 478 concepts (392 of them unique.) Here's the activity from this dataset in Neo4j:
![all connections](https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/5ae24f4e-3e71-498e-8876-46febef90709)
yellow=concepts page
blue=works page
red=item view
purple=image modal view
orange=image item view

**Comment by taceybadgerbrook** [comment]
Only a small percentage of concepts pages are followed by a link click but some concepts pages are followed by more than one link click and there are concepts that overlap and lead to the same works page. This suggests that while not all concepts are useful to users, some are serving their purpose of encouraging browsing.

Neo4j is not able to represent duplicates, so the fact that 10 of the concepts in the graph were actually viewed a total of 50 times supports this.

**Comment by taceybadgerbrook** [comment]
Concepts pages were launched with the intention of encouraging casual users to browse the collection. To understand the type of user who views concepts pages but never clicks on any of the links 100,000 records for the period 19-21/3/24 were analysed. The graph below shows the total number of actions (search, works page view, concepts page view etc) by anonymousId:![image](https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/b4665992-01ff-4ef9-9d8f-9b4f2bed1521)
Here is the same graph but showing only users with 5 or more actions:
![image](https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/d229910a-214c-45b4-8972-bb98a7bda7b0)

**Comment by taceybadgerbrook** [comment]
These don't look like casual users to me.

**Comment by taceybadgerbrook** [comment]
The graph below shows the types of actions for the users with the 10 highest totals:
![image](https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/642080da-91f4-4918-a7e8-e3c408c4a463)

**Comment by taceybadgerbrook** [comment]
@LaurenFBaily 
## Some observations and hypotheses: ##

- Reasonable numbers of users are viewing concepts pages (2,783 users viewed concepts pages 4,862 times.) -> Users are curious about the concepts pages
- Only 7 users out of a total of 2,783 users searched just once, selected a works page, viewed a concepts page and didn't continue to use the site. -> Only a small percentage of users are viewing concepts pages and terminating their session.
- There is a sizeable number of users who use the website extensively but view concepts pages and don't click on any links. -> Professional researchers are stumbling onto concepts pages. 

**Maybe casual researchers aren't getting to concepts pages because you need to search, select a work, scroll down the page and select a link before you can get view a concepts page.**

**Comment by taceybadgerbrook** [comment]
Here are the top 10 concepts pages viewed in the last 5 weeks:

<img width="786" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/09d762c6-9384-4862-886e-881887256f14">

**Comment by taceybadgerbrook** [comment]
Even counting by anonymousIds which roughly equate to GA4 users, Kibana seems to be counting significantly more users than GA4:
<img width="863" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/3fec5029-8a2a-49ac-9763-b5f0523feb98">
<img width="524" alt="image" src="https://github.com/wellcomecollection/wellcomecollection.org/assets/32418070/23c68e7f-df70-4002-8ba8-b5fcc1fe213a">

**Comment by taceybadgerbrook** [comment]
My hunch is that GA4 is counting fewer users due to users who decline cookies using the Google Analytics Opt-out Browser Add-on. @kenoir would these users still show up in Segment data?

**Comment by taceybadgerbrook** [comment]
I think the issue with the previous graphs is double counting between concept ids. When the user count per conceptId is summed, there is no adjustment for those users who viewed more than one concept page.

To judge whether there is an issue with GA4 processing or Kibana data, I selected concept pages viewed during the period 1-27 May 2024. To avoid the issue of double counting, I narrowed this to on those users who viewed https://wellcomecollection.org/concepts/t9kz4e5p.
